tutorials/en_us/PLUGINS/FAST_CHUNK_PREGENERATOR.md

```diff
@@ -0,0 +1,59 @@
+# How to install and use [Fast Chunk Pregenerator](https://www.spigotmc.org/resources/fast-chunk-pregenerator.74429/)
+
+## What is Fast Chunk Pregenerator?
+
+Fast Chunk Pregenerator, or FCP, is a plugin that generates chunks of your world so it doesn't have to generate new chunks while people are exploring. It helps with server lag because generating new chunks takes up a lot of server resources. 
+
+## Prerequisites
+
+You need a server running which can handle Spigot plugins (for example Spigot, Paper or Purpur; look [here](../../../info/en_us/SERVER_JARS.md) for a further description of the servers)
+
+Now what you need to do is download FCP. Go to [the Spigot Page](https://www.spigotmc.org/resources/fast-chunk-pregenerator.74429/) and click on `Download Now`
+
+[![Image of the Spigot Page](../../../IMAGES/PLUGINS/FAST_CHUNK_PREGENERATOR/DOWNLOAD_EXAMPLE.PNG)](https://www.spigotmc.org/resources/fast-chunk-pregenerator.74429/)
+
+## Installation
+
+Move (or copy) the downloaded JAR into the plugins directory in your server directory (where the Server JAR is).
```

Comment:
Make it clear that its the `/plugins/` folder. "plugins directory" is a bit vague.

Reply:
so
"Move (or copy) the downloaded JAR into the plugins directory `/plugins/` in your server directory (where the Server JAR is)."?